tfjs-backend-webgpu/src/kernels/reduce_webgpu.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {backend_util, util} from '@tensorflow/tfjs-core';
+import {ReduceInfo} from '@tensorflow/tfjs-core/src/ops/reduce_util';
```

Comment:
Add a TODO comment above this line to use backend_util.reduce_util with the next release of tfjs-core.

Reply:
Sorry, I am not very familiar with the TF.js project at the moment and made a lot of mistakes. I have made some changed as requested.